src/CountryFlag/index.js

```diff
@@ -1,34 +1,69 @@
 // @flow
 import * as React from "react";
 import styled from "styled-components";
+import convertHexToRgba from "@kiwicom/orbit-design-tokens/lib/convertHexToRgba";
 
 import defaultTokens from "../defaultTokens";
 import { baseURL, CODES } from "./consts";
 
 import type { Props } from "./index";
 
-const StyledCountryFlag = styled.img`
-  height: ${({ theme }) => theme.orbit.heightCountryFlag};
+const StyledCountryFlag = styled.div`
+  position: relative;
+  height: 13px; // TODO: change token heightCountryFlag
```

Comment:
shouldn't be even number ? to keep design grid ?

Reply:
It comes just from the resized image (24x13.5, 48x27). Because `13.5px` renders as `14px`, it needs to be `13px`, otherwise the `CountryFlag` is ugly.

Reply:
https://monosnap.com/file/CK0RLphbHE5ko6eQhj15dbMNZ8NEDv

for me is better 14px, 13px looks stretched